item.py

```diff
@@ -29,7 +30,7 @@ class BaseItem:
     """
 
     def __init__(self, item_name: str):
-        self.stats = ALL_ITEM_STATS[item_name].copy()
+        self.stats = Stats(ALL_ITEM_STATS[item_name].copy())
         self.passive = ItemPassive()
```

Comment:
self.passive doesn't really need to be initialized, does it ?

Reply:
yeah this passive thing needs an overhaul as here we only talk about stat passive but there are other passives that provides additional dmg